datafusion/core/src/catalog/mod.rs

```diff
@@ -295,6 +296,118 @@ impl CatalogProvider for MemoryCatalogProvider {
     }
 }
 
+/// Resolve all table references in the SQL statement.
+///
+/// ## Example
+///
+/// ```
+/// use datafusion_sql::parser::DFParser;
+/// use datafusion::catalog::resolve_table_references;
+///
```

Comment:
I think the example will format nicer if you add # in front of the use (which hides them from the docs)
```suggestion
/// # use datafusion_sql::parser::DFParser;
/// # use datafusion::catalog::resolve_table_references;
```